### DISCUSSION TITLE
A way to cancel out a shake() override

### DISCUSSION THREAD

**Discussion by UnPolloLoco** [replit/kaboom#734] [Ideas]
The title is confusing, but just read this it'll make sense... hopefully.

If you were to do a `shake(100)` and then shortly after do `shake(10)`, then the larger shake would be stopped completely. In its place would be the smaller shake.

What I think would be a great idea is if there was _an option to make `shake()` not do anything if a more powerful one is already happening._ It should override an existing shake based on how powerful the shake is NOW, not when it was called (because it gets weaker).

**Comment by slmjkdbtl** [comment]
Good point, I wonder if we should make it noop or add that 10 to the 100 that's happening.

**Reply by UnPolloLoco** [reply]
I think all these variations should be different options instead of one and only one way
